### DISCUSSION TITLE
ShinyHost.Resolve error in migrating Xamarin app to MAUI

### DISCUSSION THREAD

**Discussion by AngeloLazzari** [shinyorg/shiny#1504] [Q&A] [answered]
Hi, we are migrating the old xamarin app to MAUI and we imported this nuget 
`<PackageReference Include="Shiny.Core" Version="3.3.3" />
<PackageReference Include="Shiny.Framework" Version="4.1.0" />
<PackageReference Include="Shiny.Hosting.Maui" Version="3.3.3" />
<PackageReference Include="Shiny.Jobs" Version="3.3.3" />
<PackageReference Include="Shiny.Notifications" Version="3.3.3" />`
but event with this, when we have the command 
`ShinyHost.Resolve<IJobManager>().RunTask........... `
visual studio show us an error like: 
The name "ShinyHost" doesn't not existes in the current context.

where I am doing the mistake? 
Thank you very much!!!

**Comment by aritchie** [comment, marked as answer]
ShinyHost doesn't exist anymore

```csharp
Shiny.Hosting.Host.GetService<IJobManager>();
```

**Reply by AngeloLazzari** [reply]
Thank you very much!!!

**Reply by NiranjanShirke22** [reply]
Thanks alot this resloved my one issue. but I am also facing one more error. I have following code

 ```
using Shiny;
public class Startup :ShinyStartup
    {
        public override void ConfigureServices(IServiceCollection services, IPlatform platform)
        {
         services.RegisterJob(typeof(DownloadService))
         }
     }  
```
but now visual studio says ShinyStartup this namespace could not found. when I am accesing this in migrated maui app.

**Reply by aritchie** [reply]
@NiranjanShirke22 please use the docs and samples to see how registration is done now: https://shinylib.net/client/appbuilder/
